amundsen_application/static/js/components/SearchPage/SearchPanel/index.tsx

```diff
@@ -1,82 +1,26 @@
 import * as React from 'react';
-import { connect } from 'react-redux';
+
 import { ResourceType } from 'interfaces/Resources';
-import { GlobalState } from 'ducks/rootReducer';
-import { DashboardSearchResults, TableSearchResults, UserSearchResults } from 'ducks/search/types';
-import { TABLE_RESOURCE_TITLE, USER_RESOURCE_TITLE } from 'components/SearchPage/constants';
-import AppConfig from 'config/config';
+import ResourceSelector from 'components/SearchPage/ResourceSelector';
 
-import './styles.scss';
 
-export interface StateFromProps {
-  selectedTab: ResourceType,
-  tables: TableSearchResults;
-  dashboards: DashboardSearchResults;
-  users: UserSearchResults;
-}
+import './styles.scss';
 
-export interface OwnProps {
-  onTabChange: (tab: ResourceType) => void;
+export interface SearchPanelProps {
+  onChange: (tab: ResourceType) => void;
 }
 
-export type SearchPanelProps = StateFromProps & OwnProps;
-
-export class SearchPanel extends React.Component<SearchPanelProps> {
+export default class SearchPanel extends React.Component<SearchPanelProps> {
```

Comment:
nit: This will be more appropriate to be a React.SFC instead of a full component, but let's see what this ends up looking like before we make the call. Mind adding a `TODO: Change to React.SFC`?